src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -2,11 +2,22 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { find, filter, get, startsWith, map, orderBy } from 'lodash';
+import { find, filter, get, map, orderBy } from 'lodash';
 
 class SideNav extends Component {
   static propTypes = {
-    navItemsLookup: PropTypes.object.isRequired,
+    navItemsLookup: PropTypes.objectOf(PropTypes.shape({
```

Comment:
Hey @ncksllvn, apparently `PropTypes.objectOf` would work here so we can be more specific regarding the `navItem` shape! 😃